projects/03app_dotbot/03app_dotbot.c

```diff
@@ -19,17 +19,22 @@
 #include "board.h"
 #include "device.h"
 #include "lh2.h"
-#include "protocol.h"
 #include "motors.h"
 #include "radio.h"
-#include "rgbled.h"
 #include "timer.h"
+#include "timer_hf.h"
+// Include DRV hearders
+#include "ekf.h"
+#include "protocol.h"
+#include "rgbled.h"
+#include "ism330.h"
 
 //=========================== defines ==========================================
 
 #define DB_LH2_UPDATE_DELAY_MS    (100U)   ///< 100ms delay between each LH2 data refresh
 #define DB_ADVERTIZEMENT_DELAY_MS (500U)   ///< 500ms delay between each advertizement packet sending
 #define DB_TIMEOUT_CHECK_DELAY_MS (200U)   ///< 200ms delay between each timeout delay check
+#define DB_EKF_PREDICT_DELAY_US   (20000U) ///< 20ms delay between each EKF prediction
```

Comment:
You could just use ms directly

```suggestion
#define DB_EKF_PREDICT_DELAY_MS   (20U) ///< 20ms delay between each EKF prediction
```